src/nopoll_helpers.c

```diff
@@ -55,9 +55,14 @@ void sendMessage(noPollConn *conn, void *msg, size_t len)
 {
     int bytesWritten = 0;
     static int connErr=0;
+    nopoll_bool conn_ok;
+    nopoll_bool conn_ready;
 
     ParodusInfo("sendMessage length %zu\n", len);
-    if(nopoll_conn_is_ok(conn) && nopoll_conn_is_ready(conn))
+    conn_ok    = nopoll_conn_is_ok(conn);
+    conn_ready = nopoll_conn_is_ready(conn);
+
+    if(conn_ok && conn_ready)
```

Comment:
Since nopoll_conn_is_ready has mutex call internally so better would be to keep the way it was earlier.